Update _modeling3d.scss

Vsevolod1001 opened the pull request:
более адаптивная моб версия для экранов меньше 320рх

Vsevolod1001 committed:
Update _modeling3d.scss

более адаптивная моб версия для экранов меньше 320рх

Sunwart commented on src/sass/layout/_modeling3d.scss:
```diff
@@ -76,6 +77,10 @@
 }
+.modeling-icon{
+    width: 152.83px;
+    max-height: 133.88px;
```
Повпливало на відображення десктопної версії
![123](https://user-images.githubusercontent.com/83410878/129698320-72c26dd0-7980-4c41-b518-188b1f54f143.PNG)
Прошу виправти і відправити в поточну гілку

Sunwart reviewed (changes requested):
max-height: 133.88px; класу modeling-icon
Повпливало на відображення десктопної версії
![123](https://user-images.githubusercontent.com/83410878/129698817-2a5af761-86b6-46df-a8d8-d9e26951df05.PNG)

Прошу виправти і відправити в поточну гілку

Vsevolod1001 committed:
Update _modeling3d.scss

исправил проблему с картинками, и заменил подключение фонового изображение - теперь оно идет через after...
проверь пожалуйста, а то у меня сервер показывал дичь

Sunwart reviewed (approved):
Дякую
